examples/combined_ODNP.py

```diff
@@ -86,25 +76,43 @@
     # shut off microwave right away), but AG notes that doing so causes an
     # error.  Therefore, debug the root cause of the error and remove it!
     retval_thermal = p.dip_lock(
-        uw_dip_center_GHz - uw_dip_width_GHz / 2,
-        uw_dip_center_GHz + uw_dip_width_GHz / 2,
+        parser_dict['uw_dip_center_GHz'] - parser_dict['uw_dip_width_GHz'] / 2,
+        parser_dict['uw_dip_center_GHz'] + parser_dict['uw_dip_width_GHz'] / 2,
     )
     p.start_log()
     p.mw_off()
+    #{{{dummy scan for b12 to settle otherwise uw bleed through
+    dummy_Ep = run_spin_echo(
```

Comment:
testing this to ensure the thermal scan of the Ep has 0 microwaves leaking through